sortinghat/core/api.py

```diff
@@ -1021,35 +1023,35 @@ def _find_identities(uuids):
             if uuid == '':
                 raise InvalidValueError(msg="'uuid' cannot be an empty string")
 
-            uid = find_identity(uuid)
-            identities.append(uid)
+            identity = find_identity(uuid)
+            identities.append(identity)
 
         return identities
 
     def _set_destination_identity(trxl, identity):
```

Comment:
Should this be renamed to `_set_destination_individual`?

Reply:
I renamed it to `_set_destination_for_identity`.